documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -0,0 +1,560 @@
+// This module is included in the following files:
+//
+// assembly-client-config.adoc
+
+[id='con-broker-config-properties-{context}']
+= Kafka broker configuration tuning
+
+[role="_abstract"]
+Use configuration properties to optimize the performance of Kafka brokers.
+You can use standard Kafka broker configuration options, except for properties managed directly by Strimzi.
+
+== Basic broker configuration
+Certain broker configuration options are managed directly by Strimzi, driven by the `Kafka` custom resource specification:
+
+* `broker.id` is the ID of the Kafka broker
+* `log.dirs` are the directories for log data
+* `zookeeper.connect` is the configuration to connect Kafka with ZooKeeper
+* `listener` exposes the Kafka cluster to external clients
```

Comment:
Maybe "external" could be misleading here? External to what? Kubernetes? Clients can even run on the same Kubernetes cluster and connecting through listeners. I would just avoid to use "external".